README.md

```diff
@@ -50,33 +61,7 @@ All enums should live in their own code file, except in cases where the enum is
 
 #### Usage of MARK ####
 
-To help organizing your files you may want to use pragma marks to clearly separate your functions, properties and extensions. Only use them for the groups defined above. For extensions, use one MARK per extension. For example, `// MARK: UITableViewDelegate` instead of `// MARK: Extensions`
-
-```swift
-// MARK: Functions
-
-override func viewDidLoad() {
-}
-
-public func doSomething() {
-}
-
-internal func doSomething() {
-}
-
-private func doSomethingElse() {
-}
-
-// MARK: UITableViewDelegate Extension
-
-extension UIViewController: UITableViewDelegate {
-}
-
-// MARK: UITableViewDataSource Extension
-
-extension UIViewController: UITableViewDataSource {
-}
-```
+To help organizing your files you may want to use pragma marks to clearly separate your functions, properties and extensions. For extensions, use one MARK per extension. For example, `// MARK: UITableViewDelegate` instead of `// MARK: Extensions`.
```

Comment:
Also let's add a note about TODOs and FIXMEs since xcode lists them along with MARKs
http://nshipster.com/swift-documentation/